ephios/modellogging/templates/modellogging/statement.html

```diff
@@ -1,14 +1,15 @@
 {% load logentries %}
 {% load i18n %}
-{% translate "<em>None</em>" as NONE %}
 {{ statement.label }}{% if statement.verb %} {{ statement.verb }}{% endif %}:
 {% if statement.objects %}
     {% for obj in statement.objects %}
         {{ obj|linkify_absolute_url }}{% if not forloop.last %}, {% endif %}
     {% endfor %}
 {% else %}
-    {% if statement.old_value %}
-        {{ statement.old_value|linkify_absolute_url|default_if_none:NONE }} →
-    {% endif %}
-    {{ statement.value|linkify_absolute_url|default_if_none:NONE }}
+    {% with NONE='<i class="far fa-minus-square text-body-secondary"></i>' %}
```

Comment:
i'm not sure if i like the icon/if it is clear what it means. and it at least needs to be aligned with the arrow 
![image](https://github.com/user-attachments/assets/e90ce46e-7cb1-4505-a7a5-11b5a27e8d79)

Reply:
I think visually the whole list isn't good. I can revert the icon change for now until we do a bigger redesign.